crates/bevy_render/src/view/window/screenshot.wgsl

```diff
@@ -0,0 +1,14 @@
+@vertex
+fn vs_main(@builtin(vertex_index) in_vertex_index: u32) -> @builtin(position) vec4<f32> {
```

Comment:
I'm not sure exactly what this is doing, could you maybe add a comment explaining it?